tech/languages/csharp/dotnet-installation.md

```diff
@@ -8,26 +8,31 @@ order: 3
 
 ## Fedora .NET SIG repository
 
-.NET Core is still work in progress. The download location below contains packages that do not meet the official Fedora requirements yet (such as containing prebuilt binaries or bundled dependencies.) Please be patient, we should have reasonable packages within a few months.
+.NET Core is still work in progress. The download location below contains packages that do not meet the official Fedora requirements yet (such as using prebuilt tools to build the source) but are open source. Please be patient, we should have reasonable packages within a few months.
 
 [Fedora .NET SIG copr](https://copr.fedorainfracloud.org/coprs/g/dotnet-sig/dotnet)
 
 To enable the .NET SIG's copr repository:
 
 `$ sudo dnf copr enable @dotnet-sig/dotnet`
 
-## .NET Core SDK
+### .NET Core SDK
 
 .NET Core SDK will enable building and publishing C# source code.
 
 `$ sudo dnf install dotnet-sdk-2.0`
```

Comment:
Triple backticks are advised for code in our [contribution guide](http://developer-portal.github.io/contributing/write-general-writing-hints.html).

Reply:
Isn't it supposed to be just a header level? I don't even understand what's the meaning of it. I need two H levels on top of the title. H1 being the page title, H2 being the category (microsoft / sig) and H3 being what the package is... The structure would make no sense if both were H2. What should I use instead, bold text?...

Reply:
note the difference of 

`this inline ` 

and 

```
this
block
```

Reply:
Ref: https://help.github.com/articles/creating-and-highlighting-code-blocks/